packages/jest-cli/src/runBenchmark.js

```diff
@@ -0,0 +1,75 @@
+
+const Runtime = require('jest-runtime');
```

Comment:
There is a lot of duplication in this file. Is there a way we could re-use most of what already exists in runTest?